package.json

```diff
@@ -5,37 +5,36 @@
   "scripts": {
     "bootstrap-logkeeper": "./scripts/bootstrap-logkeeper.sh",
     "build-storybook": "storybook build",
-    "build:beta": "env-cmd -e beta -r env/.cmdrc.json yarn build",
-    "build:local": "env-cmd -e local -r env/.cmdrc-local.json yarn build",
-    "build:prod": "env-cmd -e prod -r env/.cmdrc.json yarn build",
-    "build:staging": "env-cmd -e staging -r env/.cmdrc.json yarn build",
+    "build:beta": "env-cmd -e beta yarn build",
+    "build:local": "env-cmd -e local -r .env-cmdrc.local.json yarn build",
+    "build:prod": "env-cmd -e prod yarn build",
+    "build:staging": "env-cmd -e staging yarn build",
     "build": "GIT_SHA=`git rev-parse HEAD` vite build",
     "check-types": "tsc -p tsconfig.json --noEmit",
     "codegen": "graphql-codegen --config codegen.yml",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "cy:test": "cypress run --spec",
-    "deploy-prod:do-not-use-directly": "yarn build:prod && BUCKET=evergreen.spruce yarn deploy; yarn upload-source-maps;",
-    "deploy:beta": ". ./scripts/app-version.sh && yarn build:beta && BUCKET=evergreen-beta.spruce yarn deploy",
-    "deploy:prod": "node scripts/deploy-prod.js",
-    "deploy:staging": ". ./scripts/app-version.sh && yarn build:staging && BUCKET=evergreen-staging.spruce yarn deploy",
-    "deploy": "aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete --exclude env/.cmdrc.json",
-    "dev": "env-cmd -e devLocal -r env/.cmdrc-local.json yarn start",
+    "deploy:beta": "yarn build:beta && env-cmd -e beta yarn deploy:do-not-use",
+    "deploy:staging": "yarn build:staging && env-cmd -e staging yarn deploy:do-not-use",
+    "deploy:prod": "node scripts/deploy-production.js",
+    "deploy:do-not-use": ". ./scripts/deploy.sh",
+    "dev": "env-cmd -e devLocal -r .env-cmdrc.local.json yarn start",
     "eslint:fix": "yarn eslint:strict --fix",
     "eslint:staged": "STRICT=1 eslint",
     "eslint:strict": "STRICT=1 eslint '*.{js,ts,tsx}' 'src/**/*.ts?(x)' 'cypress/**/*.ts' 'src/gql/**/*.graphql'",
-    "notify-email": "env-cmd -e prod -r env/.cmdrc.json bash email.sh",
+    "notify-email": "env-cmd -e prod bash email.sh",
```

Comment:
Might be able to delete this command, I can't find any references to it and Parsley doesn't have one
```suggestion
```